WebRoot/templates/admin/Widgets.template

```diff
@@ -82,12 +88,18 @@
 <template id='admin.Widgets#ZaBaseDialog' width='20' height='32'>
 	<div class='DwtDialog WindowOuterContainer'>
 		<table role="presentation">
-			<tr class='DwtDialogHeader' id='${dragId}'>
+			<tr class='DwtDialogHeader' id='${dragId}' style='position:relative;'>
 				<td class='minWidth'>${icon}</td>
 				<td id='${id}_title' class='DwtDialogTitle'>${title}</td>
 				<td class='minWidth'><div class='${closeIcon2}'></div></td>
 				<td class='minWidth'><div class='${closeIcon1}'></div></td>
 				<td width=24 id='${id}_minimize' class='DwtDialogMinIcon'></td>
+				<td id='loadingStatusBar' class='loadingContainer' style='visibility:hidden;'>
+					<div style='position:relative;height:inherit;'>
+						<div class='LoaderTitle'>Loading...</div>
```

Comment:
please don't set non-translated string in a template.

Reply:
Same as above.